core/src/main/kotlin/keiyoushi/utils/Date.kt

```diff
@@ -3,12 +3,11 @@ package keiyoushi.utils
 import java.text.ParseException
 import java.text.SimpleDateFormat
 
-@Suppress("NOTHING_TO_INLINE")
-inline fun SimpleDateFormat.tryParse(date: String?): Long {
+fun SimpleDateFormat.tryParse(date: String?): Long {
```

Comment:
Inlining normal functions doesn't provide performance boost. R8 will inline it if there's only 1 call site.